dll/win32/comctl32/static.c

```diff
@@ -627,6 +627,18 @@ static void STATIC_PaintTextfn( HWND hwnd, HDC hdc, DWORD style )
     else
     {
         DrawTextW( hdc, text, -1, &rc, format );
+        if(style & WS_DISABLED)
+        {
+            COLORREF oldTextColor=GetTextColor(hdc);
+            SetBkMode(hdc, TRANSPARENT);
+            SetTextColor(hdc, GetSysColor(COLOR_HIGHLIGHTTEXT));
+            rc.left++;rc.top++;
+            DrawTextW( hdc, text, -1, &rc, format );
+            SetTextColor(hdc, oldTextColor);
+            rc.left--;rc.top--;
+            DrawTextW( hdc, text, -1, &rc, format );
+            SetBkMode(hdc, OPAQUE);
+        }
```

Comment:
```suggestion
        UINT flags = DST_TEXT | ((style & WS_DISABLED) ? DSS_DISABLED : DSS_NORMAL);
        if (format & DT_HIDEPREFIX)
            flags |= DSS_HIDEPREFIX;

        hBrush = GetSysColorBrush(COLOR_GRAYTEXT);
        DrawStateW(hdc, hBrush, STATIC_DrawTextCallback,
                   (LPARAM)text, (WPARAM)format,
                   rc.left, rc.top,
                   rc.right - rc.left, rc.bottom - rc.top,
                   flags);
```
and you need to add _**BEFORE**_ `STATIC_PaintTextfn` the following function:
```c
/**********************************************************************
 *       STATIC_DrawTextCallback
 *
 *   Callback function used by DrawStateW function.
 */
static BOOL CALLBACK STATIC_DrawTextCallback(HDC hdc, LPARAM lp, WPARAM wp, int cx, int cy)
{
    RECT rc;

    SetRect(&rc, 0, 0, cx, cy);
    DrawTextW(hdc, (LPCWSTR)lp, -1, &rc, (UINT)wp);
    return TRUE;
}
```

Reply:
Thank you very much!
DrawStateW is right solution for better compactibility.
I see it in apimonitor in Windows, but not in ReactOs(in apimonitor).
I will be test tomorow.